lib/agent0/tests/predict_trade_test.py

```diff
@@ -0,0 +1,576 @@
+"""Test predicting the outcome of a trade."""
+
+from __future__ import annotations
+
+import logging
+from copy import deepcopy
+from decimal import Decimal
+from typing import NamedTuple
+
+import pytest
+from ethpy.hyperdrive.interface.read_interface import HyperdriveReadInterface
+from ethpy.hyperdrive.state import PoolState
+from fixedpointmath import FixedPoint
+from tabulate import tabulate
+
+from agent0.hyperdrive.interactive import InteractiveHyperdrive
+from agent0.hyperdrive.interactive.chain import Chain
+from agent0.hyperdrive.interactive.event_types import OpenLong, OpenShort
+
+# it's just a test
```

Comment:
afaik every other test in the repo adheres to these constraints, so I think this one should too